tests/test_plots.py

```diff
@@ -0,0 +1,153 @@
+import ssapy
+import os
+import shutil
+import numpy as np
+import matplotlib.pyplot as plt
+
+save_folder = os.path.expanduser('./ssapy_test_plots/')
+print(f"Putting test_plot.py output in: {save_folder}")
+
+# Testing rotate_vector() in utils.
+v_unit = np.array([1, 0, 0])  # Replace this with your actual unit vector
+
+figs = []
+
+temp_directory = f'{save_folder}rotate_vector_frames/'
```

Comment:
Please add a `/` between the `{save_folder}` and `rotate_vector_frames`, just for safety purposes. If someone changes `save_folder` I don't want this to fail because they forgot a slash.